cucumber/api/features/policy_versions_retention.feature

```diff
@@ -24,3 +24,13 @@ Feature: Limit the number of policy versions
     """
     Deleting policy version: {:version=>1, :resource_id=>"cucumber:policy:policy_test_version",
     """
+
+  Scenario: Migrate db remove extra policies
```

Comment:
do we want to also add this verification?
```
    And The following appears in the log after my savepoint:
    """
    Deleting policy version: {:version=>1, :resource_id=>"cucumber:policy:policy_test_version",
    """
```

Reply:
It shouldn't, the upgrade should not write to the log